client/scss/components/_header.scss

```diff
@@ -0,0 +1,289 @@
+    }
+  }
+}
+
+.header__burger-trigger {
+  position: absolute;
+  width: 1.3rem;
+  height: 1rem;
+  top: -2.7rem;
+
+  @include respond-to('medium') {
+    display: none;
+  }
+
+  // TODO: replace with svg-sprite icon
+  span {
+    position: absolute;
+    left: 0;
+    right: 0;
+    height: 2px;
+    background: color('black');
+    transition: transform 400ms ease;
+    transform-origin: center center;
+
+    &:nth-child(2) {
+      top: 0.4rem;
+    }
+
+    &:last-child {
+      bottom: 0;
+    }
+
+    .header__lower.is-active & {
+      top: 0.5rem;
+
+      &:first-child {
+        transform: rotate(45deg);
+      }
+
+      &:nth-child(2) {
+        display: none;
+      }
+
+      &:last-child {
+        transform: rotate(-45deg);
+        bottom: auto;
+      }
+    }
+  }
+}
+
+.header__brand {
+  flex-basis: (100% / 3);
+  display: flex;
+  justify-content: center;
+
+  @include respond-to('medium') {
+    border-right: 1px solid color('keyline-grey');
+    flex-basis: auto;
+    width: 10rem;
+  }
+}
+
+.header__search {
+  flex-basis: (100% / 3);
+  min-width: 0;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.header__button {
+  @include font('WB7');
+  appearance: none;
+  border: 0;
+  background: color('transparent');
+  padding: 0 0 0 1.5rem;
+  display: flex;
+
+  @include respond-to('medium') {
+    border-left: 1px solid color('keyline-grey');
+  }
+}
+
+// TODO: replace with svg-sprite icon
+.header__button-icon {
+  width: 1rem;
+  height: 1rem;
+  position: relative;
+  border: 2px solid color('black');
+  border-radius: 50%;
+
+  &:after {
+    content: '';
+    position: absolute;
+    transform: rotate(45deg);
+    bottom: -0.25rem;
+    right: -0.5rem;
+    display: block;
+    width: 0.75rem;
+    height: 2px;
+    background: color('black');
+  }
+}
+
+.header__button-text {
+  display: none;
+  margin-left: 1rem;
+
+  @include respond-to('medium') {
+    display: block;
+  }
+}
+
+.header__input {
+  @include font('WB7');
+  display: none;
+  margin-right: 1rem;
+  padding: 0 1rem;
+
+  .no-js &,
+  .header__search.is-active & {
+    display: block;
+  }
+}
+
+.header__logo {
+  width: 100%;
+  max-width: 8rem;
+
+  @include respond-to('medium') {
+    margin-right: 1.5rem;
+    width: auto;
+  }
+}
+
+.header__nav {
+  display: none;
+  margin-top: 0.5rem;
+
+  .no-js &,
+  .header__lower.is-active & {
+    display: block;
+  }
+
+  @include respond-to('medium') {
+    display: block;
+    margin-top: 0;
+  }
+}
+
+.header__list {
+  @include plain-list();
+  @include font('WB7');
+  margin-left: -0.3rem;
+
+  @include respond-to('medium') {
+    display: flex;
+  }
+}
+
+.header__item {
+  margin-right: 1.5rem;
+
+  @include respond-to('large') {
+    margin-right: 2rem;
+  }
+}
+
+.header__dropdown {
+  display: none;
+  position: absolute;
+  margin-top: 0;
+  top: 100%;
+  background: color('cream');
+  border-top: 1px solid color('keyline-grey');
+  padding-bottom: 2rem;
+
+  .no-js &,
```

Comment:
`no-js` would be a class added to the `html` element by default and removed by a script in the head (determining that a user's JS is working – at least at the time of page load). This is a standard pattern to determine whether or not progressively-enhanced content needs to be displayed by default. I don't think it makes sense to be BEM-ed up.